eos-update-flatpak-repos

```diff
@@ -658,9 +793,14 @@ def _migrate_installed_flatpaks():
                                          new_ostree_ref))
                     try:
                         repo.prepare_transaction()
+                        kwargs = {}
+                        if kind == Flatpak.RefKind.APP:
```

Comment:
Yes, although limited to origin migrations on runtimes, because branch or name changes would be somewhat senseless for these. And actually, the runtime migrations are pre eos3->eos3a checkpoint so this script shouldn't encounter them any longer in the FlatpakInstalledRef list, just the dict of ostree refs. We need to keep the table of migrations there to clear up leaked refs at the end as this was only fixed after the migration (also the reason this is structured as two loops rather than just doing the ostree refs as we do the flatpaks).